src/styles/common/_legend.scss

```diff
@@ -1,13 +1,29 @@
 // TODO:
 // - Review the naming
 // - Create mixins for animation
+@import "../helpers/variables";
 
 $enter-opacity: 0.97;
 $exit-opacity: .2;
 
 .britechart-legend {
 
     .legend-entry {
+        font-size: $text-size-medium;
+
+        .legend-entry-name,
+        .legend-entry-value,
+        .legend-circle {
+            // Ensure font-size can't be set on child elements
+            // Allowing font-size on parent to be used in calculations
+            font-size: 1em !important;
```

Comment:
Let's use local variables for these two values, like in those 'enter-opacity' and 'exit-opacity' examples at the top of the file